### PR TITLE
[DX-1437] customise --> customize (PR 1) 

### PR DESCRIPTION
**Pull request by Roeegg2**
### For internal users - Please add a Jira DX PR ticket to the subject!
<br>

---

### Preview Link

<br>

### Description

<br>

#### Screenshots (if appropriate)
<br>

## Checklist

- [ ] I have added a **preview link** to the PR description.
- [ ] I have reviewed the suggestions made by our AI (PR Agent) and updated them accordingly (spelling errors, rephrasing, etc.)
- [ ] I have [reviewed the guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING.md) for contributing to this repository.
- [ ] I have [read the technical guidelines](https://github.com/TykTechnologies/tyk-docs/blob/master/CONTRIBUTING-TECHNICAL-GUIDE.md) for contributing to this repository.
- [ ] Make sure you have started *your change* off *our latest `master`*.
- [ ] I **labelled** the PR

### CODE REVIEW

**Comment by Roeegg2** [comment]
1. I modified the `alt` tag data on `tyk-docs/content/configure/outbound-email-configuration.md`
2. I didn't touch `tyk-docs/content/error-response-codes.md`

File: tyk-docs/content/tyk-on-premises/installation/on-heroku.md [hunk 1]
```diff
@@ -426,7 +426,7 @@ After re-starting the Gateway, the logs should be showing something similar to t
 
 Set this variable back to an empty value in order to revert back to the default behavior.
 
-## Upgrading or Customising Tyk
+## Upgrading or Customizing Tyk
```
Comment (dcs3spp) [inline tyk-docs/content/tyk-on-premises/installation/on-heroku.md:429]: Change is ok, no linking refs affected to the anchor

File: tyk-docs/content/configure/outbound-email-configuration.md [hunk 2]
```diff
@@ -101,16 +101,16 @@ If `SMTPUsername` or `SMTPPassword` is omitted, Tyk assumes that authentication
   "SecretAccessKey": "KEY"
 },
 ```
-### Customise your Welcome Emails
+### Customize your Welcome Emails
```
Comment (dcs3spp) [inline tyk-docs/content/configure/outbound-email-configuration.md:104]: Change is ok, no linking refs affected by change to anchor

**Review by dcs3spp — COMMENTED** [review]
Change is ok, no linking refs affected by change to anchor

File: tyk-docs/content/configure/outbound-email-configuration.md [hunk 3]
```diff
@@ -123,14 +123,14 @@ You can customise the welcome email that a developer recieves when they signup t
 6. Click **Save** at the top of the Portal Settings screen.
 
 
-### Customise your Key Approval Emails
+### Customize your Key Approval Emails
```
Comment (dcs3spp) [inline tyk-docs/content/configure/outbound-email-configuration.md:126]: Change is ok, no linking refs affected by change to anchor

File: tyk-docs/content/advanced-configuration/transform-traffic/response-headers.md [hunk 4]
```diff
@@ -19,7 +19,7 @@ There are related [Request Header Transform]({{< ref "transform-traffic/request-
 
 ## When to use response header transformation
 
-#### Customising responses for specific clients
+#### Customizing responses for specific clients
```
Comment (dcs3spp) [inline tyk-docs/content/advanced-configuration/transform-traffic/response-headers.md:22]: Change is ok, no linking refs affected by change to anchor

**Review by letzya — APPROVED** [review]

**Pull request merged**